lib/view/bibliotheek/BibliotheekCatalogusDatatableResponse.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace CsrDelft\view\bibliotheek;
+
+use CsrDelft\model\bibliotheek\BoekModel;
+use CsrDelft\model\entity\bibliotheek\Boek;
+use CsrDelft\model\ProfielModel;
+use CsrDelft\model\security\LoginModel;
+use CsrDelft\view\formulier\datatable\DataTable;
+use CsrDelft\view\formulier\datatable\DataTableResponse;
+
+class BibliotheekCatalogusDatatableResponse extends DataTableResponse {
+
+	/**
+	 * @param Boek $entity
+	 * @return false|string
+	 */
+	public function getJson($entity) {
+		$arr = (array) $entity;
```

Comment:
Iets transparanter en customizable (en consistent).

```suggestion
		$arr = $entity->jsonSerialize();
```